packages/extension-chakra-storefront/jest.config.js

```diff
@@ -43,11 +43,15 @@ module.exports = {
         createTestGlob('components/field'),
         createTestGlob('components/reset-password'),
         createTestGlob('components/with-registration'),
+        createTestGlob('components/social-login'),
+        createTestGlob('components/toggle-card'),
         createTestGlob('pages/home'),
         // createTestGlob('pages/login'),  // TODO: enable after Account page has been migrated
         createTestGlob('pages/login-redirect'),
         createTestGlob('pages/social-login-redirect'),
         // createTestGlob('pages/registration'), // TODO: enable after Account page has been migrated
+        '<rootDir>/src/pages/checkout/partials/contact-info.test.js',
```

Comment:
There are some errors in ContactInfo's tests, in particular for the passwordless ones.